platforms/android/app/src/main/AndroidManifest.xml

```diff
@@ -50,20 +49,10 @@
         </provider>
     </application>
 
-    <!-- Required to access Google Play Licensing -->
-    <uses-permission android:name="com.android.vending.CHECK_LICENSE" />
-    <!-- Required to download files from Google Play -->
-    <uses-permission android:name="android.permission.INTERNET" />
-    <!-- Required to keep CPU alive while downloading files (NOT to keep screen awake) -->
-    <uses-permission android:name="android.permission.WAKE_LOCK" />
-    <!-- Required to poll the state of the network connection and respond to changes -->
-    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-    <!-- Required to check whether Wi-Fi is enabled -->
-    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE"/>
+
     <!-- Required to read and write the expansion files on shared storage. -->
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
Remove this

Reply:
Removed. WRITE_EXTERNAL_STORAGE implies READ_EXTERNAL_STORAGE permission.